### PR TITLE
Feature/127

### PR DESCRIPTION
**Pull request by Reivs**
Added Game Over and Level Score screen

### CODE REVIEW

File: Assets/01_Scripts/LevelManager.cs [hunk 1]
```diff
@@ -15,4 +18,28 @@ public class LevelManager : MonoBehaviour
...
+    public void DecreaseLives() { lives -= 1; }
+
+    public void GameIsOver() {
+       if(lives <= 0) {
+
+            sceneLoader.gameIsOver = true;
+            Debug.Log("game is over");
+        }
+        
+            }
```
Comment (Knayd) [inline Assets/01_Scripts/LevelManager.cs]: Could you fix the indent of this curly brace?

File: Assets/01_Scripts/SceneLoader.cs [hunk 2]
```diff
@@ -9,20 +9,28 @@ public class SceneLoader : MonoBehaviour
...
 
-    
+    private float timeCountdownBetweenScreens = 3f;
 
     public int getLocalSceneIndex;
+    
 
     void Start()
     {
+        //gameIsOver = false;
```
Comment (Knayd) [inline Assets/01_Scripts/SceneLoader.cs]: Could you remove this comment?

File: Assets/01_Scripts/SceneLoader.cs [hunk 3]
```diff
@@ -9,20 +9,28 @@ public class SceneLoader : MonoBehaviour
     [SerializeField] GameObject levelCanvas;
     [SerializeField] GameObject highScoreCanvas;
     [SerializeField] GameObject pauseMenuCanvas;
+    [SerializeField] GameObject GameOverCanvas;
+    [SerializeField] GameObject LevelScoreCanvas;
+    private LevelManager levelManager;
+    public bool gameIsOver = false;
 
-    
+    private float timeCountdownBetweenScreens = 3f;
```
Comment (Knayd) [inline Assets/01_Scripts/SceneLoader.cs]: Could you rename this to `countdownTimeBetweenScreens`? I think it makes more sense grammar wise

File: Assets/01_Scripts/LevelManager.cs [hunk 4]
```diff
@@ -6,7 +6,10 @@ public class LevelManager : MonoBehaviour
 {
     [SerializeField] private int score = 0;
     [SerializeField] private int lives = 3;
+    
 
+   public SceneLoader sceneLoader;
```
Comment (Knayd) [inline Assets/01_Scripts/LevelManager.cs]: Could you make this variable `private`? It's only being used inside this script

File: Assets/01_Scripts/SceneLoader.cs [hunk 5]
```diff
@@ -9,20 +9,28 @@ public class SceneLoader : MonoBehaviour
     [SerializeField] GameObject levelCanvas;
     [SerializeField] GameObject highScoreCanvas;
     [SerializeField] GameObject pauseMenuCanvas;
+    [SerializeField] GameObject GameOverCanvas;
+    [SerializeField] GameObject LevelScoreCanvas;
+    private LevelManager levelManager;
```
Comment (Knayd) [inline Assets/01_Scripts/SceneLoader.cs]: Could you remove this `levelManager` variable? It is only being initialized on line 25 but it's never used after that

**Review by Knayd — CHANGES_REQUESTED** [review]

File: Assets/01_Scripts/SceneLoader.cs (hunk 3 above)
Comment (Reivs) [reply Assets/01_Scripts/SceneLoader.cs]: I didn't like how tall that capital 'T' was...

File: Assets/01_Scripts/LevelManager.cs [hunk 6]
```diff
@@ -4,15 +4,58 @@
 
 public class LevelManager : MonoBehaviour
 {
+    [SerializeField] GameObject GameOverCanvas;
+    [SerializeField] GameObject LevelScoreCanvas;
```
Comment (Knayd) [inline Assets/01_Scripts/LevelManager.cs]: The name of these variables (`GameOverCanvas` and `LevelScoreCanvas`) should start with lower case, could you change them?

File: Assets/01_Scripts/LevelManager.cs [hunk 7]
```diff
@@ -4,15 +4,58 @@
...
+
     [SerializeField] private int score = 0;
     [SerializeField] private int lives = 3;
 
+    private float countdownTimeBetweenScreens = 3f;
 
+    private SceneLoader sceneLoader;
```
Comment (Knayd) [inline Assets/01_Scripts/LevelManager.cs]: Could you remove this variable? Is only getting initialized on line 33 but never used afterwards

**Review by Knayd — CHANGES_REQUESTED** [review]

File: Assets/01_Scripts/LevelManager.cs (hunk 6 above)
Comment (Reivs) [reply Assets/01_Scripts/LevelManager.cs]: No


K I will

**Review by Knayd — APPROVED** [review]

**Pull request merged**